README.md

```diff
@@ -22,7 +22,6 @@ All packages are published separately as NPM packages, and should be installed a
 
 ## Packages
 
-- [cra-template-edu](https://github.com/chanzuckerberg/frontend-libs/tree/main/packages/cra-template-edu)
```

Comment:
Technically people could still actually use this template, since the package is out there in the npm registry.